Update sample_13TeV_25ns.json

ashrafkasem opened the pull request:
adding GluGluZZ samples and also modifying the dtag for them to be compatible with skimlist output to avoid any problem in merging 

https://docs.google.com/spreadsheets/d/1iPdZEm_jpvOnFchXasV3lJo2Oe1yYvSLUD7FSRD6KjI/edit#gid=1594342584
https://cmsweb.cern.ch/das/request?view=list&limit=150&instance=prod%2Fglobal&input=dataset%3D%2FGluGluToContinToZZTo*_13TeV_MCFM701_pythia8%2FRunIISpring16MiniAODv2-PUSpring16_80X_mcRun2_asymptotic_2016_miniAODv2_v0-v1%2FMINIAODSIM

ashrafkasem committed:
Update sample_13TeV_25ns.json

adding GluGluZZ samples and also modifying the dtag for them to be compatible with skimlist output to avoid any problem in merging 

https://docs.google.com/spreadsheets/d/1iPdZEm_jpvOnFchXasV3lJo2Oe1yYvSLUD7FSRD6KjI/edit#gid=1594342584
https://cmsweb.cern.ch/das/request?view=list&limit=150&instance=prod%2Fglobal&input=dataset%3D%2FGluGluToContinToZZTo*_13TeV_MCFM701_pythia8%2FRunIISpring16MiniAODv2-PUSpring16_80X_mcRun2_asymptotic_2016_miniAODv2_v0-v1%2FMINIAODSIM

AndreasAlbert commented on DMAnalysis/data/sample_13TeV_25ns.json:
```diff
@@ -97,6 +122,54 @@
           ]
+        },
+        {
+          "dtag": "MC13TeV_ggToZZTo2e2mu",
```
Could you also change this to "GluGluToZZ"? Just for consistency.

AndreasAlbert commented on DMAnalysis/data/sample_13TeV_25ns.json:
```diff
@@ -97,6 +122,54 @@
+          ]
+        },
+        {
+          "dtag": "MC13TeV_ggToZZTo2e2tau",
```
gg -> GluGlu

ashrafkasem committed:
Update sample_13TeV_25ns.json

your requested modifications have been done 
by the way the 2 samples GluGluToZZTo2e2mu and GluGluToZZTo2e2tau are no produced yet

AndreasAlbert commented:
Closes #7
